data/ignition/files/usr/local/share/assisted-service/configmap.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config
+data:
+  ASSISTED_SERVICE_HOST: 127.0.0.1:8090
+  ASSISTED_SERVICE_SCHEME: http
+  AUTH_TYPE: none
```

Comment:
If the productized version requires secure communication between the agent and assisted-service, AUTH_TYPE will need to be updated. For now as a POC, this is ok.